buyer-app/buyer-app/src/state/entities/downloadData/helpers.js

```diff
@@ -0,0 +1,16 @@
+import Config from "../../../config";
+import authorization from "../../../utils/headers";
+
+const apiUrl = Config.get("api.url");
+
+const getData = async orderId => {
+  const res = await fetch(`${apiUrl}/download-data/${orderId}`, {
+    headers: {
+      Authorization: authorization()
+    }
+  });
+
+  return res.ok ? res.text() : res.json();
```

Comment:
I think this specific return is not very clear, if it fails I return a json, if not a text? I would handle the error in a different way, maybe returning an error message?